lib/Pod/To/Text.pm6

```diff
@@ -113,9 +113,12 @@ sub declarator2text($pod) {
 
 sub signature2text($params) {
       $params.elems ??
-      "(\n\t" ~ $params.map({ $_.perl }).join(", \n\t") ~ "\n)" 
+      "(\n\t" ~ $params.map(&param2text).join("\n\t") ~ "\n)" 
       !! "()";
 }
+sub param2text($p) {
+    $p.perl ~ ',' ~ ( $p.WHY ?? ' # ' ~ $p.WHY !! ' ')
```

Comment:
Concatenating a comma like this  will result in possible hanging commas unlike the previous method of using `join`